src/openforms/contrib/kvk/validators.py

```diff
@@ -48,6 +52,12 @@ class KVKBranchNumberValidator(NumericBaseValidator):
 validate_branchNumber = KVKBranchNumberValidator()
 
 
+def get_kvk_search_results(query: SearchParams) -> dict:
+    with get_kvk_search_client() as client:
+        result = client.get_search_results(query_params=query)
+        return result
```

Comment:
```suggestion
        return client.get_search_results(query_params=query)
```

Reply:
I don't agree with these cosmetic change suggestions, one form or the other really doens't matter, but the current form is much more `breakpoint()` friendly.

Reply:
True, even though `__return__` exists in pdb